sched/clock/clock_systime_ticks.c

```diff
@@ -154,3 +154,25 @@ clock_t clock_systime_ticks(void)
 # endif /* CONFIG_SYSTEM_TIME64 */
 #endif /* CONFIG_SCHED_TICKLESS */
 }
+
+/****************************************************************************
+ * Name:  clock_systime_steps
+ *
+ * Description:
+ *   Add system time by idle ticks.
+ *
+ * Input Parameters:
+ *   ticks - Idle ticks
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+void clock_systime_steps(sclock_t ticks)
+{
+  irqstate_t flags;
+  flags = enter_critical_section();
+  g_system_timer += ticks;
```

Comment:
If enable CONFIG_SCHED_TICKLESS, g_system_timer cannot be used as system tick.

Reply:
Done

Reply:
I see this PR is still in draft status, could you change it so that it can be merged into the NuttX mainline, thank you.

Reply:
I have changed this PR into ready status for merge.

Reply:
LGTM